### ISSUE TITLE
Create securedrop-svs-disp-config package

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-workstation#182]
To configure disposable VMs used for viewing documents, we need to create a Debian package that will do the following:

- install the package dependencies outlined in [this spreadsheet](https://docs.google.com/spreadsheets/d/1l-DGR5UnnvYA5jfeZtplER933ANPWgFLx_uWbnJZKWI/edit#gid=0)
- configure the appropriate MIME type handlers system-wide via `/usr/share/applications/mimeapps.list` as documented in #158.

Aside from basic verification, we should ensure that PDFs don't open in LibreOffice but in `evince` (by default, LibreOffice Draw will claim PDF files after installation).

**Comment by conorsch** [comment]
Added Salt configuration logic to install this package, and it definitely installs cleanly. PR incoming. In order to test the opening in DispVMs, I'll need to submit more documents to the shared test instance (since we recently rotated the submission key used for testing). Doing that now.

In the near future, we should upload some curated submissions with many different filetypes, to aid in interactive testing during review.
